app/assemblers/assemble_contributors.rb

```diff
@@ -1,7 +1,9 @@
 class AssembleContributors
   include Mandate
 
-  initialize_with :params
+  initialize_with params: Mandate::KWARGS
+
+  def self.keys = %i[track_slug user_handle page category period]
```

Comment:
I needed to add this as otherwise I could not use `**` with Rails `params` in the calling controllers. This error was shown:

> ActionController::UnfilteredParameters: unable to convert unpermitted parameters to hash